jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/screen/trigger/trigger-model-edit.xml

```diff
@@ -27,6 +27,9 @@
                 <comboBox id="scheduleTypeField" property="scheduleType" required="true" nullOptionVisible="false"/>
                 <textField id="cronExpressionField" property="cronExpression" required="true" visible="false"
                            contextHelpTextHtmlEnabled="true" contextHelpText="msg://cronExpressionHelpText"/>
+                <radioButtonGroup id="repeatModeSelector" optionsEnum="io.jmix.quartzui.screen.trigger.RepeatMode"
+                                  visible="false" caption="msg://triggerRepeatMode"
+                                  contextHelpText="msg://triggerRepeatModeHelpText" contextHelpTextHtmlEnabled="true"/>
                 <textField id="repeatCountField" property="repeatCount" visible="false"/>
```

Comment:
what about adding additional context help near the repeatCount field as well? We could explain one more time that "total number of firings will be this number + 1". Users may not read the context help near the radio group.

Reply:
Done